main.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const table = document.querySelector('table');
+const tbody = document.createElement('tbody');
+table.appendChild(tbody);
+const errorInfo = document.querySelector('div#errorinfo');
+
+function getUser(users, userId) {
+  return users.find((user)=> {
+    return user.id === userId;
+  });
+}
+
+function fillTable() {
+  let request = new XMLHttpRequest();
+  request.open('GET', 'https://jsonplaceholder.typicode.com/users', false);
```

Comment:
do not use synchronous HTTP requests, them blocks user interactions with the page